modules/contentbox/models/comments/Comment.cfc

```diff
@@ -31,9 +31,9 @@ component
 	property
 		name     ="commentID"
 		fieldtype="id"
-		generator="native"
-		setter   ="false"
-		params   ="{ allocationSize = 1, sequence = 'commentID_seq' }";
+		column="id"
```

Comment:
I prefer to stay away from using the same column name of `id` in all entities.  Especially when dealing with ambiguity.  I prefer to stick to the actual name as chosen.

Reply:
I get that. I think it's a bit "old school" and complicates this migration to GUIDs a great deal - I will have to double the code to re-loop all of the foreign keys and add the references only _after_ the PK column has been dropped and renamed.  I can do this, and I figured you might have an objection to this. ;) 

Speaking from personal experience, I really believe that, while you might run in to ambiguity on the hibernate side of things ( which can easily be fixed by specificity ), the ability to abstract your code, when every entity has the same name, more than makes up for the database ambiguity.

Reply:
FWIW, here's an example of how easy it is to prep an an entity for population from a Vue component, when all of the PKs are the same, by preflighting it through `Vue.options.filters.persistableEntity( this.formData )`

```
var persistable = {
    /**
	* Creates a persistable array from a collection of objects
	**/
    persistableArray( collection ){
		
        if( collection.results ){
            return collection.results;
        }

        var ids = [];
        for( var i in collection ){
            if( collection[ i ] !== null ) {
                if( collection[ i ].id ){				
                    ids.push( collection[ i ].id );	
                } else {
                    ids.push( collection[ i ] );
                }
            }
        }
        // always de-dupe the assignments
        return [ ...new Set( ids ) ];
    },

    /**
	* Creates a persistable object ( e.g. a GUID string ) from an object
	**/
    persistableObject( rel ){

        if( !rel ) return rel;
	
        //if it is already a string or we don't know what to do with it - return the original object
        if( typeof( rel ) === "string" ) return rel;
		
        return rel.id || null;

    },

    /**
	* Preflights an entity for persistence
	**/
    persistableEntity( entity, exclude ){

        //duplicate so we don't modify references
        var objData = Object.assign( {}, entity );
        var objKeys = Object.keys( objData );

        for( var i in objKeys ){

            var key = objKeys[ i ];

            if( objData[ key ] && typeof( objData[ key ] )  === "object" ){
                if( Array.isArray( objData[ key ] ) || objData[ key ].results ){
                    objData[ key ] = this.persistableArray( objData[ key ] );
                } else {
                    objData[ key ] = this.persistableObject( objData[ key ] );
                }
            }

        }

        if( exclude ){
            for( var t in exclude ){
                delete( objData[ exclude[ t ] ] );
            }
        }

        return objData;
    }

};

export default persistable;
```